contracts/SignerBondsUniswapV2.sol

```diff
@@ -103,9 +107,13 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
         );
     }
 
-    /// @notice Swaps signer bonds.
-    /// @dev Adds incoming bonds to the overall contract balance.
-    function swapSignerBonds() external payable override {}
+    /// @notice Receive ETH upon withdrawal of risk manager's signer bonds.
+    /// @dev Do not send arbitrary funds. They will be locked forever.
+    receive() external payable {}
+
+    /// @notice Notifies the strategy about signer bonds purchase.
+    /// @param amount Amount of purchased signer bonds.
+    function onSignerBondsPurchased(uint256 amount) external override {}
```

Comment:
Is this left intentionally empty?

Reply:
https://github.com/keep-network/coverage-pools/pull/88#discussion_r650672931